getting_started/intel_debug/introduction.rst

```diff
@@ -137,4 +137,65 @@ drivers and should work when changing the ``dsp_driver`` parameter.
 The selection of firmware, topology, and UCM files is based on platform
 capabilities, codec names, and DMI options. While the SOF team and the
 community try to cover all possible cases, errors will happen when the
-wrong file is selected at any of the three layers.
\ No newline at end of file
+wrong file is selected at any of the three layers.
+
+4. Chromebooks and SOF
+----------------------
+
+As stated above, starting from 2019/2020 Chromebook firmwares have been
```

Comment:
I know this is under intel debug, but might want to reinforce this is explicitly Intel chromebooks for this date. Just a thought so a random user who ended up here via a search didnt grab a QC chromebook and was very confused when there was no SOF onboard

Reply:
Done.